az-digital/az_quickstart
# Closes #2385 Add indent and outdent to az_standard text editor

## Related issues
Closes #2385

Turns out, in order to enable indenting of content that is not a list, you need to download and install a contrib module.

https://www.drupal.org/project/ckeditor_indentblock

## How to test
Edit any content that uses the az_standard text formatter.
See two new buttons in the alignment section for indent and outdent.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
